### PR TITLE
added note on import order

### PR DESCRIPTION
**Pull request by prayogoa**

### CODE REVIEW

File: Dependencies.md [hunk 1]
```diff
@@ -16,3 +16,14 @@ Always ask yourself, do we really need to include that dependency. Ask a colleag
 Some dependencies support submodules. This means that even though the entire library is installed, you can opt to only include a tiny part of it. This helps keep our footprint small.
 
 For example, to include lodash in a module, you'd usually do something like `import _ from 'lodash';`. But since lodash consists of many tiny modules, you should just get the modules you need for this file, like `import pick from 'lodash/object/pick'`.
+
+### Import order
+When importing, put third party modules first, followed by our own modules ordered by distance from the current file
+```
+import React from 'react'
+import isEmpty from 'lodash/lang/isEmpty'
+import config from 'app/config';
```
Comment (macocha) [inline Dependencies.md]: empty line between the two kinds helps with readability

File: Dependencies.md [hunk 2]
```diff
@@ -16,3 +16,14 @@ Always ask yourself, do we really need to include that dependency. Ask a colleag
...
+import { TEST_API_URL } from 'app/constants';
+import dateHelper from 'app/utilities/date';
+import { fetchTests } from './actions';
+```
```
Comment (jkaipr) [inline Dependencies.md]: you can add styles at the bottom, then an empty line and some example type import
```js
import type { Map } from 'immutable';
```

**Comment by pscohn** [comment]
Can we add a rule to only import at the very top of the file? I've noticed some imports hidden away near the middle/bottom of files at times that made it harder to find when updating filenames.

Also we could add a link to this file from the readme.

**Comment by ziahamza** [comment]
Agreed to only have imports at top 👍👍👍

**Pull request merged**